ignite/metrics/loss.py

```diff
@@ -42,40 +42,40 @@ class Loss(Metric):
         and the Loss metric using an ``evaluator`` created with
         :meth:`~ignite.engine.create_supervised_evaluator` method.
 
-        .. code-block:: python
+        .. testcode::
 
-            import torch
-            import torch.nn as nn
-            from torch.nn.functional import nll_loss
-
-            from ignite.metrics import Accuracy, Loss
-            from ignite.engine import create_supervised_evaluator
-
-            model = ...
+            model = nn.Linear(10, 3)
```

Comment:
Have a look to this [file](https://github.com/pytorch/ignite/blob/master/docs/source/conf.py). A default model is already defined and it would be great to use this one rather than define a new one.